pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -965,7 +972,7 @@ func readLineToEvent(r *logfileReader, logChan chan<- logEntry) {
 			level, err := log.ParseLevel(loginfo.Level)
 			if err != nil {
 				log.Errorf("ParseLevel failed: %s\n", err)
-				level = log.DebugLevel
+				level = log.InfoLevel
```

Comment:
I don't know what impact this will have on the other log files in /persist/IMGx/log/
Can you check if there is a noticable increase in the amount of log traffic from a real device with this change?

Reply:
Majority of log files present in /persist/IMGx/log are created/logged by logrus itself. Because logrus creates our log message, loglevel parsing should not fail.
When I enable container logs (onboot) I see some logs coming with "level: err", "level: notify" which logrus fails to parse. Since there are error type messages also being discarded (default log level set to Info), I had to make the above change.

Reply:
Can you add a bug on that in the tracker so we can handle those two levels?

Reply:
Added the following story in tracker.
https://www.pivotaltracker.com/story/show/170579610